libs/common-services/src/test/java/uk/gov/di/ipv/core/library/service/CiMitUtilityServiceTest.java

```diff
@@ -153,9 +154,17 @@ void getMitigationJourneyResponseShouldReturnMitigationWhenCiCanBeMitigated() th
         // arrange
         var code = "ci_code";
         var journey = "some_mitigation";
-        var ci = ContraIndicator.builder().code(code).issuanceDate("some_date").build();
+        String document = "doc_type/213123";
```

Comment:
done

Reply:
Sorry, I more meant a test that succesfully finds the mitigation route for a CI that isn't document related.

Reply:
So I think something like this might be the missing test:
```
    @Test
    void getMitigationJourneyResponseShouldReturnMitigationWhenCiCanBeMitigateWithNoDoc() throws Exception {
        // arrange
        var code = "ci_code";
        var journey = "some_mitigation";
        var ci =
                ContraIndicator.builder()
                        .code(code)
                        .issuanceDate("some_date")
                        .build();
        var cis = ContraIndicators.builder().contraIndicatorsMap(Map.of(code, ci)).build();
        when(mockConfigService.getCimitConfig())
                .thenReturn(Map.of(code, List.of(new MitigationRoute(journey, null))));
        Map<String, ContraIndicatorConfig> ciConfigMap =
                Map.of(code, new ContraIndicatorConfig(code, 7, -5, "X"));
        when(mockConfigService.getContraIndicatorConfigMap()).thenReturn(ciConfigMap);
        when(mockConfigService.getSsmParameter(CI_SCORING_THRESHOLD)).thenReturn("5");

        // act
        var result = ciMitUtilityService.getCiMitigationJourneyStep(cis);

        // assert
        assertEquals(Optional.of(new JourneyResponse(journey)), result);
    }

```

Reply:
Added ...coverage was not showing that.